ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -804,6 +804,59 @@ public func timer(interval: NSTimeInterval, onScheduler scheduler: DateScheduler
 	}
 }
 
+/// Creates a repeating timer of the variable interval starting with `nextInterval(0)`,
+/// with a reasonable default leeway, sending updates on the given scheduler.
```

Comment:
There is also `public func variableTimer(onScheduler scheduler: DateSchedulerType, withLeeway leeway: NSTimeInterval, nextInterval: NSTimeInterval -> NSTimeInterval?) -> SignalProducer<NSDate, NoError>` function, so `leeway` is already parametrized.

But I'm wondering if we should accept fixed leeway for variable timer intervals. Maybe replacing to `leewayFactor` (default = 0.1) will be better.